### PR TITLE
fix(DB/Quest): Missing Event on questId (1020, 1033, 1034, 1035)

### PR DESCRIPTION
**Pull request by r0m1ntik**
## Changes Proposed:
-  Add Event for Quest: [1020](https://www.wowhead.com/quest=1020/orendils-cure), [1033](https://www.wowhead.com/quest=1033/elunes-tear), [1034](https://www.wowhead.com/quest=1034/the-ruins-of-stardust), [1035](https://www.wowhead.com/quest=1035/fallen-sky-lake)

## Issues Addressed:
- Closes [4197](https://github.com/azerothcore/azerothcore-wotlk/issues/4197)
- And 3 others quest (not reported)

## Tests Performed:

Снимок экрана 2021-01-04 в 14 55 16

## How to Test the Changes:

1) .q a 1020 or 1033 or 1034 or 1035
2) .q c 1020 or 1033 or 1034 or 1035
3) .go cr id 3894

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1609769436615146000.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+(3894, 5, 0, 'There you go...', 12, 7, 100, 0, 0, 0, 1440, 0, 'Pelturas Whitemoon'),
+(3894, 6, 0, '%s places a poultice on Relara\'s head.', 16, 0, 100, 0, 0, 0, 1443, 0, 'Pelturas Whitemoon'),
+(3894, 7, 0, '%s breathes a sigh of relief.', 16, 0, 100, 0, 0, 0, 1444, 0, 'Pelturas Whitemoon'),
+(3894, 8, 0, '%s crushes the moonstones into a cup of tea.', 16, 0, 100, 0, 0, 0, 1346, 0, 'Pelturas Whitemoon'),
+(3894, 9, 0, 'Here you are, Relara.  Drink this...', 12, 7, 100, 16, 0, 0, 1347, 0, 'Pelturas Whitemoon'),
+(3894, 10, 0, 'Relara... you\'re awake?', 12, 7, 100, 0, 0, 0, 1350, 0, 'Pelturas Whitemoon'),
+(3894, 11, 0, 'Rest now, my daughter.', 12, 7, 100, 1, 0, 0, 1351, 0, 'Pelturas Whitemoon'),
+(3894, 12, 0, '%s looks at $n, and then weeps with joy.', 16, 0, 100, 0, 0, 0, 1352, 0, 'Pelturas Whitemoon'),
+(3894, 13, 0, 'Relara is saved, $n.  Thank you.', 12, 7, 100, 18, 0, 0, 1353, 0, 'Pelturas Whitemoon');
+DELETE FROM `smart_scripts` WHERE (`entryorguid` IN (3892, 3894, 389400, 389401, 389402, 389403)) AND (`source_type` IN (0, 9));
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1609769436615146000.sql:24]: please change to:

```sql
DELETE FROM `smart_scripts` WHERE (`entryorguid` IN (3892, 3894)) AND (`source_type`= 0));
DELETE FROM `smart_scripts` WHERE (`entryorguid` IN (389400, 389401, 389402, 389403)) AND (`source_type`= 9));
```

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

**Comment by FrancescoBorzi** [comment]
thanks @r0m1ntik and @pklloveyou !

**Comment by Kitzunu** [comment]
Thanks

**Pull request merged**